app/forms.py

```diff
@@ -0,0 +1,143 @@
+# import re
+from django import forms
+# from PIL import Image
+from django.contrib.auth.models import User
+from django.utils.translation import ugettext_lazy as _
+# from django.core.exceptions import ValidationError
+import time
+
+
+class RegistrationForm(forms.Form):
+
+    username = forms.RegexField(
+        regex=r'^\w+$',
+        widget=forms.TextInput(attrs=dict(required=True, max_length=30)),
+        label=_("Username"),
+        error_messages={'invalid': _("This value must contain only letters, \
+         numbers and underscores.")})
```

Comment:
Also the `\` in the multiline string isn't necessary, since they're in brackets you can just use two strings on separate lines, they'll be combined into a single string:

```python
>>> print ("foooo"
...        "bar"
...        "baz")
foooobarbaz
```